packages/amplify-cli/src/commands/status.ts

```diff
@@ -1,13 +1,29 @@
+import { ViewResourceTableParams, CLIParams } from "amplify-cli-core";
+
+
 export const run = async context => {
-  await context.amplify.showResourceTable();
-  await context.amplify.showHelpfulProviderLinks(context);
-  await showAmplifyConsoleHostingStatus(context);
+  const cliParams:CLIParams  = { cliCommand : context?.input?.command,
+                                 cliSubcommands: context?.input?.subCommands,
+                                 cliOptions : context?.input?.options }
+
+  const view = new ViewResourceTableParams( cliParams );
+  if ( context?.input?.subCommands?.includes("help")){
+     console.log( view.getStyledHelp())
+  } else {
+    try {
+      await context.amplify.showStatusTable( view );
+      await context.amplify.showHelpfulProviderLinks(context);
+      await showAmplifyConsoleHostingStatus(context);
+    } catch ( e ){
+      view.logErrorException(e);
```

Comment:
Can we context.print.error here instead of console.log()?

Reply:
will do

Reply:
Done